### DISCUSSION TITLE
酷安开屏ad失效

### DISCUSSION THREAD

**Discussion by kumanlu** [gkd-kit/gkd#258] [General]
酷安开屏广告不会自动点击跳过，具有偶发性。具体作者自测。

**Comment by lisonge** [comment]
https://github.com/gkd-kit/subscription/issues/new?template=bug_report.yml

**Reply by lisonge** [reply]
找个加速器或者其它科学上网工具就行，或者b站搜索 GitHub 相关内容也行

此外你遇到的这个酷安开屏广告是把跳过按钮直接嵌在图片里了，无障碍无法获取准确位置
